central/scannerdefinitions/handler/handler.go

```diff
@@ -122,19 +133,21 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 func (h *httpHandler) get(w http.ResponseWriter, r *http.Request) {
 	uuid := r.URL.Query().Get(`uuid`)
 	fileName := r.URL.Query().Get(`file`)
-
 	// If only file is requested, then this is for Scanner v4.
```

Comment:
This is no longer accurate because you have a new param `version.` Can you update?